Console application for dividing the bill total among a number of payers.

Neolight79 opened the pull request:
The complete functioning console application has been developed. Please, follow the instructions in README.md

Commit:
Implemented console application for dividing the bill total among a number of payers.

ArturNurtdinov commented on src/main/java/CheckItem.java:
```diff
@@ -0,0 +1,10 @@
+public class CheckItem {
+    String itemName;
+    double value;
```
Лучше сделать их final, чтобы не было возможности поменять значение после создания объекта

Neolight79 replied:
Done

ArturNurtdinov commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,56 @@
+        System.out.println("Введите количество участников (2-999):");
+        while (true) {
+            int persons = ScanValues.scanIntegerValue();
+            if (persons > 1 && persons < 1000) {
```
А почему ты выбрал 1000 в качестве максимума? вроде такого в требованиях нет

Neolight79 replied:
Done

ArturNurtdinov commented on src/main/java/ScanValues.java:
```diff
@@ -0,0 +1,38 @@
+
+public class ScanValues {
+    public static int scanIntegerValue() {
+        Scanner scanner = new Scanner(System.in);
```
В каждой функции создаётся новый объект Scanner. Можно было бы в конструкторе этого класса создать один объект сканнера и переиспользовать его, т.к. создание нового объекта занимает время и память

Neolight79 replied:
Done

Commit:
1. Scanner moved to class fields
2. Upper limitation 1000 persons removed
3. Fields in CheckItem.java made final
4. Negative values for the goods are excluded
